sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -361,7 +362,7 @@ Command *Scheduler::GraphBuilder::addCGUpdateHost(
 }
 
 // The functions finds dependencies for the requirement. It starts searching
-// from list of "leaf" commands for the record and check if the examining
+// from list of "leaves" commands for the record and check if the examining
```

Comment:
```suggestion
// from list of "leaf" commands for the record and check if the examining
```
Singular noun as adjective is correct here.

Reply:
Fixed. Thanks!